### PR TITLE
resolve Shell-839

### PR DESCRIPTION
**Pull request by ankitgyawali**

### CODE REVIEW

**Comment by KalleV** [comment]
@ankitgyawali This PR does not implement the `/versions` API route described in SHELL-839.

File: test/lib/unit/api/api-config_spec.js [hunk 1]
```diff
@@ -69,17 +69,31 @@ describe('ApiConfig', () => {
...
          it(`will test if the GET /version route assigned by api/api-config.js is working fine`, function (done) {
-           request.get("/api-package-1-namespace/version")
+           request.get("/version")
                 .expect(200)
                 .then(res => {
-                    expect(JSON.parse(res.text).name).toBe("api-package-1");
-                    expect(JSON.parse(res.text).version).toBe('0.0.1');
+                    res.text = res.text ? JSON.parse(res.text) : {};
+                    expect((res.text[0]).api).toBe("main-package");
+                    expect((res.text[0]).version.version).toBe('0.0.1');
```
Comment (KalleV) [inline test/lib/unit/api/api-config_spec.js]: @ankitgyawali Why is the version property a nested value? The response data does not appear to be correct.

**Comment by KalleV** [comment]
@ankitgyawali It seems like the response data will contain entries for API services that do not expose any routes. Could you filter out LabShare packages that do not contain any routes please?

**Comment by ankitgyawali** [comment]
@KalleV filter them out completely or just add a property denoting that information? i thought the idea was to expose the whole dependency list

**Comment by KalleV** [comment]
@ankitgyawali Filter them out completely. If a particular LabShare API package doesn't expose any routes, there's no reason to display it in the response.

**Pull request merged**